sdk/keyvault/azure-security-keyvault-certificates/src/samples/java/com/azure/security/keyvault/certificates/CertificateAsyncClientJavaDocCodeSnippets.java

```diff
@@ -332,11 +332,11 @@ public void updateCertificatePolicyCodeSnippets() {
             .subscribe(certificatePolicyResponseValue -> {
                 CertificatePolicy certificatePolicy = certificatePolicyResponseValue;
                 // Update transparency
-                certificatePolicy.setCertificateTransparency(true);
+                certificatePolicy.setCertificateTransparent(true);
                 certificateAsyncClient.updateCertificatePolicy("certificateName", certificatePolicy)
```

Comment:
not using the updated value?
here and below.

Reply:
updated value comes from the subscription and gets used.